cmd/flightctl-pam-issuer/main.go

```diff
@@ -0,0 +1,132 @@
+//go:build linux
+
+package main
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/flightctl/flightctl/internal/api_server/middleware"
+	"github.com/flightctl/flightctl/internal/config"
+	"github.com/flightctl/flightctl/internal/crypto"
+	"github.com/flightctl/flightctl/internal/instrumentation"
+	"github.com/flightctl/flightctl/internal/pam_issuer_server"
+	"github.com/flightctl/flightctl/pkg/log"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	ctx := context.Background()
+
+	log := log.InitLogs()
+	log.Println("Starting PAM issuer service")
+	defer log.Println("PAM issuer service stopped")
+
+	cfg, err := config.LoadOrGenerate(config.ConfigFile())
+	if err != nil {
+		log.Fatalf("reading configuration: %v", err)
```

Comment:
If we log fatal defer statment will never fire above. Cleaner to have a constructor return err and log that fatal
```
if err := RunCmd(ctx); err != nil {
    	log.Fatalf("pam issuer: %v", err)
}
```